components/identity-mgt-new/org.wso2.carbon.identity.mgt/src/main/java/org/wso2/carbon/identity/mgt/dto/UserIdentityClaimsDO.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * 
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.mgt.dto;
+
+import org.wso2.carbon.identity.mgt.store.UserIdentityDataStore;
+import org.wso2.carbon.user.core.UserCoreConstants;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.TreeMap;
+
+/**
+ * This encapsulates the user's data that is related user's login information
+ */
+public class UserIdentityClaimsDO implements Serializable {
```

Comment:
Data object cannot be in dto package. Place it in model package. Remove the "DO" part from the name.